Jenkinsfile_parameterized

```diff
@@ -0,0 +1,20 @@
+#!groovy
+@Library("Infrastructure") _
+
+properties([
+		parameters([
+			string(name: 'PRODUCT_NAME', defaultValue: 'platform', description: ''),
```

Comment:
although very minor one could we please keep the alignment consistent.Easier to read?
```parameters([
 			string(name: 'PRODUCT_NAME', defaultValue: 'platform',      description: ''),
 			string(name: 'APP',          defaultValue: 'job-scheduler', description: ''),
 			string(name: 'TYPE', 		     defaultValue: 'java',          description: ''),
 			string(name: 'ENVIRONMENT',  defaultValue: 'sandbox',       description: 'Environment where code should be build and deployed'),
 			choice(name: 'SUBSCRIPTION', choices: 'sandbox\nnonprod\nprod', description: 'Azure subscriptions available to build in')
 		])
 ])```

Reply:
yeh definitely - thanks 👍